docs/monitoring/02-01-monitoring.md

```diff
@@ -41,7 +41,7 @@ The complete monitoring flow in Kyma comes down to these components and steps:
 ![](./assets/monitoring-architecture.svg)
 
 1. Upon Kyma installation on a cluster, Prometheus Operator creates a Prometheus instance with default configuration.
-2. The Prometheus server periodically polls all metrics exposed on endpoints of services specified in Service Monitors. Prometheus stores these metrics in a time-series database.
-3. If Prometheus detects any anomalies in metrics that are covered by alert rules, it triggers the alerts and passes them to Alertmanager.
+2. The Prometheus server periodically polls all metrics exposed on endpoints of services specified in ServiceMonitor CRDs. Prometheus stores these metrics in a time-series database.
```

Comment:
What do you think about? 
> The Prometheus server periodically scrapes for metrics from all the `/metrics` endpoints of the services specified in ServiceMonitor CRDs.

Reply:
I will just stick to the "poll" verb but I changed the rest :)

Reply:
And I changed "services" into" pods"